testing/src/main/java/org/oppia/android/testing/espresso/TextInputAction.kt

```diff
@@ -0,0 +1,56 @@
+package org.oppia.android.testing.espresso
+
+import android.view.View
+import androidx.test.espresso.matcher.BoundedMatcher
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+
+/**
+ * Action for matching error text and performing other actions on TextInputLayout
+ * in a test infrastructure-specific way.
+ */
+class TextInputAction {
+
+  companion object {
+    /**
+     * Check that the TextInputLayout view has the [expectedErrorText] error text.
```

Comment:
```suggestion
     * Checks that the TextInputLayout view has the [expectedErrorText] error text.
```

(note verb tense). Ditto for the other method.

Reply:
Done.